core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -335,4 +452,35 @@ public final boolean isTlsEnabled() {
     public final void setTlsEnabled(boolean enabled) {
         this.tlsEnabled = enabled;
     }
+
+    /**
+     * Returns an instance of {@link ProxyOptions} configured with the proxyHost, proxyPort, proxyUsername and
```

Comment:
no need to mention nor link to the returned type. JavaDoc handles this automatically.